### PR TITLE
Helios Station

### PR DESCRIPTION
**Pull request by noctyrnal**
## About the PR

Adds a new high pop station with a cosy and luxury design to it.

## Why / Balance

N/A

## Technical details

N/A

## Media

Helios-0

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A

**Changelog**

add: Added Helios Station

### CODE REVIEW

**Comment by noctyrnal** [comment]
I forgot to add it to the map pools, oopsies.

**Comment by IamVelcroboy** [comment]
You'll also need to add the map to this [file](https://github.com/DeltaV-Station/Delta-v/blob/master/Content.IntegrationTests/Tests/PostMapInitTest.cs)

**Comment by UnicornOnLSD** [comment]
From the looks of it, very nice station.
Few maints but nicely done, epic work my man, love perma 👍

**Comment by Timemaster99** [comment]
Will the people sitting on the court spectator benches be able to see or hear the judge? They seem quite a bit far away

File: Resources/Prototypes/Maps/helios.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  minPlayers: 75
+  stations:
+    Helios:
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Helios {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'TG'
```
Comment (DebugOk) [inline Resources/Prototypes/Maps/helios.yml]: Is this a TG station?

Comment (noctyrnal) [reply Resources/Prototypes/Maps/helios.yml]: I forgot to change the prefix to 14, let me go fix that now.

Comment (IamVelcroboy) [reply Resources/Prototypes/Maps/helios.yml]: You know, I've never bothered changing the prefix. We should probably make it DV on all our maps...?

**Comment by IamVelcroboy** [comment]
Noctis informed me they won't be maintaining this.

**Comment by UnicornOnLSD** [comment]
Hello, hi, yes, I will be taking over Helios and doing requested changes until it's considered polished enough, I shall open a PR once I've done a decent bit of work

**Pull request closed without merging**